src/rules/semicolonRule.ts

```diff
@@ -357,8 +357,8 @@ class SemicolonNeverWalker extends SemicolonWalker {
     }
 
     private checkVariableStatement(node: ts.VariableStatement) {
-        const declarations = node.declarationList.declarations;
-        if (declarations[declarations.length - 1].initializer === undefined) {
+        const declaration = last(node.declarationList.declarations);
+        if (declaration === undefined || declaration.initializer === undefined) {
```

Comment:
Declaring a new `last<T>` is a bit of overhead for this. How about:

```typescript
if (declarations.length !== 0 && declarations[declarations.length - 1].initializer === undefined)
```

Reply:
🙂 not particularly. [`tslint.json`](https://github.com/palantir/tslint/blob/master/tslint.json) has it set to 140 characters, so this shouldn't trigger it.